Sources/TuistKit/Utils/TuistVersionLoader.swift

```diff
@@ -13,6 +13,8 @@ final class TuistVersionLoader: TuistVersionLoading {
     }
 
     func getVersion() throws -> String {
-        try system.capture(["tuist", "version"])
+        try system
+            .capture(["tuist", "version"])
+            .replacingOccurrences(of: "\n", with: "")
```

Comment:
You can do `spm_chomp()` which is a more robust version of removing trailing newline chars.

Reply:
done